<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaborSelect</class>
 <widget class="QDialog" name="LaborSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>531</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Select Job</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QcjDataTable" name="jobSelectTable">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;This table is for associating a job with a workorder. It's xml form name is &lt;em&gt;job_select_table&lt;/em&gt;&lt;/html&gt;</string>
        </property>
        <property name="xml_definition" stdset="0">
         <string>job_select_table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="addJobBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add to Workorder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="doneBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Workorder Jobs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QcjDataFrame" name="laborFrame">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="xml_definition" stdset="0">
          <string>form_definition</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QcjDataForm" name="laborForm">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;This form is for modifying jobs associated with a workorder. It's xml form name is &lt;em&gt;wo_job_form&lt;/em&gt;&lt;/html&gt;</string>
            </property>
            <property name="xml_definition" stdset="0">
             <string>wo_job_form</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QcjDataTable" name="laborTable">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;This table is for viewing and selecting jobs associated with a workorder. It's xml form name is &lt;em&gt;wo_job_table&lt;/em&gt;&lt;/html&gt;</string>
            </property>
            <property name="xml_definition" stdset="0">
             <string>wo_job_table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QPushButton" name="saveLaborBtn">
               <property name="text">
                <string>Save Job</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteLaborBtn">
               <property name="text">
                <string>Remove Job</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Associated Parts</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QcjDataFrame" name="partsFrame">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="xml_definition" stdset="0">
          <string>form_definition</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QcjDataForm" name="partsForm">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;This form is for modifying parts included in a jobs associated with a workorder. It's xml form name is &lt;em&gt;wo_job_part_form&lt;/em&gt;&lt;/html&gt;</string>
            </property>
            <property name="xml_definition" stdset="0">
             <string>wo_job_part_form</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QcjDataTable" name="partsTable">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;This table is for viewing and selecting parts included in a jobs associated with a workorder. It's xml form name is &lt;em&gt;wo_job_part_form&lt;/em&gt;&lt;/html&gt;</string>
            </property>
            <property name="xml_definition" stdset="0">
             <string>wo_job_part_table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QPushButton" name="savePartsBtn">
               <property name="text">
                <string>Save Part</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deletePartsBtn">
               <property name="text">
                <string>Remove Part</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcjDataForm</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataForm.h</header>
  </customwidget>
  <customwidget>
   <class>QcjDataFrame</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QcjDataTable</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>doneBtn</sender>
   <signal>clicked()</signal>
   <receiver>LaborSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
